json-lint/schemas/metrics/providers/googleAnalytics.json

```diff
@@ -8,7 +8,8 @@
         "additionalProperties": false,
         "properties": {
             "name": {
-                "description": "Name of the metrics provider",
+                "description": "Name of the metrics provider only letters and numbers",
+                "pattern": "^[a-zA-Z][a-zA-Z\\d]*$",
```

Comment:
this regex doesn't match `t0`, which is the name example given by google 😬 
(I think it should be `\d` instead of `\\d`)

Is there a reason why you're forbidding numbers at the beginning of the name ? I don't think they're forbidden 🤔

Reply:
Oh I thought it was like variables that the first one can't be a number. I will remove that then.
Yes, it should be `\d` but as it's inside and string, we need to scape it.
![image](https://user-images.githubusercontent.com/33353095/58175640-e039b280-7ca0-11e9-8a2a-956def32985c.png)
Like this ones:
![image](https://user-images.githubusercontent.com/33353095/58175715-14ad6e80-7ca1-11e9-95f2-f615fa458918.png)

Reply:
Oooooh I see!

For starting with numbers, I honestly don't know. I couldn't find more info online about it. Let's try to find specifications again before making a decision.